doc/reference/storage_zfs.md

```diff
@@ -115,17 +115,19 @@ Key                           | Type                          | Default
 
 Key                     | Type      | Condition                 | Default                                        | Description
 :--                     | :---      | :--------                 | :------                                        | :----------
+`block.filesystem`      | string    | block-based driver        | same as `volume.block.filesystem`              | {{block_filesystem}}
```

Comment:
I think this is all good now, except for this part "block-based driver", as ZFS can have both block-based and non-block based volumes. Really these settings are dependent on a volume also have `zfs.block_mode=true` on them. Without that they are not appropriate for a ZFS volumes. So how about we add the condition here as `zfs.block_mode volume` or something like that?